src/components/profile/profile-logged-in/adopted-trees.tsx

```diff
@@ -4,66 +4,104 @@ import { TertiaryButton } from "../../buttons/tertiary";
 import { TreeCard } from "./tree-card";
 import { useAuthStore } from "../../../auth/auth-store";
 import { AdoptedTreeIcon } from "../../icons/adopted-tree-icon";
+import { Skeleton } from "../../skeleton/skeleton";
+
+export interface AdoptedTreesWrapperProps {
+	sectionTitle?: string;
+}
+
+export const AdoptedTreesWrapper: React.FC<
+	AdoptedTreesWrapperProps & React.HTMLAttributes<HTMLDivElement>
+> = ({ sectionTitle, ...props }) => {
+	return (
+		<div className="md:shadow-gdk-soft mb-3 md:rounded-2xl md:border-2 md:p-7">
+			<h2 className="text-2xl font-semibold">{sectionTitle}</h2>
+			{props.children}
+		</div>
+	);
+};
```

Comment:
i think this component + props interface should be in its own file 👍  Also tiny naming suggestion, I think `AdoptedTreesCard` would be a easier to understand, wrapper can be a bit abstract.